app/src/org/commcare/CommCareNoficationManager.java

```diff
@@ -63,7 +63,7 @@ private void updateMessageNotification() {
             // The PendingIntent to launch our activity if the user selects this notification
             Intent i = new Intent(context, MessageActivity.class);
 
-            PendingIntent contentIntent = PendingIntent.getActivity(context, 0, i, 0);
+            PendingIntent contentIntent = PendingIntent.getActivity(context, 0, i, PendingIntent.FLAG_IMMUTABLE);
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/636ed8f68dcb7d308025e7f6c07ce9c3f8729d63/app/src/org/commcare/CommCareNoficationManager.java#L66"></span>Field requires API level 23 (current min is 16): <code>android.app.PendingIntent#FLAG_IMMUTABLE</code></td>
    </tr>
  </tbody>
</table>